src/common/config.cc

```diff
@@ -604,15 +604,19 @@ int md_config_t::parse_option(std::vector<const char*>& args,
 
   if (ret != 0 || !error_message.empty()) {
     if (oss) {
-      *oss << "Parse error setting " << option_name << " to '"
-           << val << "' using injectargs";
+      if(option_name && !val.empty()) {
```

Comment:
under which circumstance, will `option_name` be null? and i think it's fine to print `val` even if it's empty.